app/models/cash_transfer.rb

```diff
@@ -106,4 +107,8 @@ class CashTransfer < Ekylibre::Record::Base
   def opened_financial_year?
     FinancialYear.on(transfered_at)&.opened?
   end
+
+  def financial_year_in_closure_preparation?
```

Comment:
This doesn't make it clear when's the financial year. Maybe something like `transferred_during_closure_prepration?` instead?